manga_auto_translator/main.py

```diff
@@ -1,12 +1,16 @@
-from argparse import ArgumentParser
```

Comment:
(OPTIONAL) Same comment : FYI : ArgumentParser is fine, if you want to make clean client script, you can use click package which is better -> https://click.palletsprojects.com/en/8.1.x/